### ISSUE TITLE
The "id" argument must be of type string

### ISSUE THREAD

**Issue by iamwangkj** [VladimirMikulic/parcel-plugin-custom-dist-structure#25]
**Describe the bug**
When I run build script, it say: The "id" argument must be of type string.

**How it happened?**
I want to moving static resources to a separate folder. And when I run build script, it say: The "id" argument must be of type string.

**Desktop (please complete the following information):**
 - OS: macOS
 - Parcel version: v1.12.4
 - Plugin version: v1.1.12

**Plugin configuration object:**
```
"customDistStructure": {
    "config": {
      ".js": "js",
      "img": [
        ".jpg",
        ".png"
      ],
      "css": [".scss", ".css"]
    },
    "options": {
      "development": true
    }
  }
```

**Comment by VladimirMikulic** [comment]
Hi @iamwangkj. Thank you for reporting this issue. Could you please specify project folder structure (src) and how files are referenced?

**Comment by VladimirMikulic** [comment]
@iamwangkj please check out v1.1.13
The error should be gone :)

**Comment by EdgarKogel** [comment]
Hi! I have the same problem and  FIX #26 does not help in my case. 

If you install Parcel globally as mentioned on the Parcel Getting Started page (`yarn global add parcel-bundler`) this error is still there. But the fix helped me to find a workaround for now.

If I add `parcel-bundler` also locally with `yarn add -D parcel-bundler` the plug-in worked for me like expected and helped me to achieve the output structure I want. So thanks for writing this plug-in.

Maybe it's possible to cover the global installation also in future versions, because this seems to be the standard way of installing Parcel at least for new users.

**Comment by VladimirMikulic** [comment]
Hi @EdgarKogel. Thank you for your comment.
I am glad my plugin served you well.

Unfortunately, it's a bit complicated to retrieve the path of the globally installed Parcel package.
As a fellow JS developer, you probably understand the fragmentation of this ecosystem :)
Some users use NPM, others use yarn, some use `parcel` package, then others use `parcel-bundler` and so on...

I agree with you that this is mostly a standard way of installing Parcel, but covering all of the above cases is error-prone. (already tried once)
Until further notice, you'll need to have Parcel as a project's development dependency.
